content/applications/hr/payroll/salary_attachments.rst

```diff
@@ -0,0 +1,146 @@
+  :guilabel:`Total Amount` field is populated, this date is updated.
+- :guilabel:`Document`: If any documentation is needed, such as a court order, click the
+  :guilabel:`Upload your file` button and a file explorer window loads. Select the desired document
+  to attach it to the record. Only **one** document can be attached to a salary attachment.
+- :guilabel:`Monthly Amount`: Enter the amount taken out of each paycheck every month.
+- :guilabel:`Total Amount`: This field **only** appears if the :ref:`salary attachment type
+  <salary-attachment/attachment-types>` has no end date (the :guilabel:`No End Date` option is
+  **not** ticked.)
+
+.. image:: salary_attachments/salary-attachment-form.png
+   :alt: The salary attachment form with all fields filled out.
+
+Since the salary attachment form autosaves as the fields are populated, after making a salary
+attachment for an individual employee, there is no further action required.
+
+If creating salary attachments for multiple employees on a single salary attachment form, after the
+form is filled out, click the :guilabel:`Create Individual Attachments` button. This creates
+separate salary attachments for each of the employees listed in the :guilabel:`Employees` field.
+
+After the separate salary attachments have been created, the screen returns to the :guilabel:`Salary
+Attachment` dashboard, but with a :guilabel:`Description` filter populated with the description
+filled in on the salary attachment form. All the salary attachments have a status of
+:guilabel:`Running`, since they are currently active. Clear the filter in the search box to view the
+default salary attachment dashboard in its entirety.
+
+Manage salary attachments
+=========================
+
+Salary attachments can have one of three statuses: *running*, *completed*, or *canceled*. To view
+the current status of all salary attachments, navigate to :menuselection:`Payroll app --> Contracts
+--> Salary Attachments`.
+
+All salary attachments appear in the order they were configured. To view the salary attachments by
+a particular metric, such as the :guilabel:`Status`, or :guilabel:`Type`, click on the column title
+to sort by that specific column.
+
+Completed salary attachments
+----------------------------
+
+When a salary attachment is created, it has a status of :guilabel:`Running`. Once the salary
+attachment is finished (the :guilabel:`Total Amount` entered on the :ref:`salary attachment form
+<salary-attachment/new-attachment>` has been paid in full), the status automatically changes to
+:guilabel:`Completed`, and the employee no longer has the money taken out of future paychecks.
+
+If a salary attachment has been fulfilled, but has not automatically changed to
+:guilabel:`Completed`, the record can be manually updated. To change the status, open the
+:guilabel:`Salary Attachment` dashboard by navigating to :menuselection:`Payroll app --> Contracts
```

Comment:
```suggestion
*Salary Attachment* dashboard by navigating to :menuselection:`Payroll app --> Contracts
```
using italics here because it is *followed* by the directions on how to get there, so if you'd like to use a GUI, place it after the menuselection